app/Models/WebhookConfiguration.php

```diff
@@ -150,4 +167,41 @@ public static function discoverCustomEvents(): array
 
         return $events;
     }
+
+    /**
+     * @param  array<mixed, mixed>  $replacement
+     * @return array<mixed, mixed>|string|null
+     * */
+    public function replaceVars(array $replacement, string $subject): array|string|null
```

Comment:
Switch the return type to either ?array or ?string but not both. Make two methods if needed.